src/quick-lint-js/lex.h

```diff
@@ -108,6 +108,11 @@ class lexer {
   // Postcondition: this->peek().type == token_type::regexp.
   void reparse_as_regexp();
 
+  // Returns true if a valid regexp literal comes after '*/' or '**/'
+  // Precondition: this->input[offset+1] == '/'
+  // offset is number of characters preceding the final /
```

Comment:
These comments don't agree with each other. If `input_` points to `"**/x"`, then the 2 characters precede the `/`. However, `this->input[2+1] == 'x'`.